tests/unit/lexicon/test_engine.py

```
@@ -0,0 +1,247 @@
+                    grimoire=grimoire,
+                    channel=_channel(),
+                )
+            )
+        return grimoire
+
+    @staticmethod
+    def _finished(grimoire: Grimoire) -> list[RiteFinished]:
+        return [e for e in grimoire.events if isinstance(e, RiteFinished)]
+
+    @classmethod
+    def test_a_step_that_raises_still_closes_its_rite(cls):
```

Ruff (0.15.21)

[warning] Missing return type annotation for classmethod `test_a_step_that_raises_still_closes_its_rite`

Add return type annotation: `None`

(ANN206)